spec/javascripts/health-care/utils/validations.spec.js

```diff
@@ -0,0 +1,25 @@
+import Validations from '../../../../_health-care/_js/_utils/validations.js';
```

Comment:
I don't expect to ever have such a class. :P

`import Validations from 'blah'`  conceptually means
`const Validations = getDefaultExportFrom(blah);`

I can name the import whatever I want.